src/components/structures/UserSettings.js

```diff
@@ -1233,10 +1233,10 @@ module.exports = React.createClass({
                             value={this.presentableTextForThreepid(val)} disabled
                         />
                     </div>
-                    <div className="mx_UserSettings_threepidButton mx_filterFlipColor">
+                    <AccessibleButton className="mx_UserSettings_threepidButton mx_filterFlipColor">
```

Comment:
I'm not sure this one will work since the `onClick` is on the `img` rather than the `div`?

Reply:
Thanks for alerting me to this. It turns out I have added more issues like this. I can fix almost all of them with upload avatar button being more tricky. So I'm leaving that out for today.